### ISSUE TITLE
哪一个损失函数最好用，损失函数类中里面又不带参数？

### ISSUE THREAD

**Issue by jingzhengli** [CoinCheung/pytorch-loss#25]
你好，我看到am-softmax这个损失函数里面带着参数需要更新，这个参数实际上就是分类器的最后一层是吧。我来请教一下哪种损失函数比较容易上手呢？同时效果还比较好？

**Comment by CoinCheung** [comment]
maybe you can start from nn.crossentropyloss.
